packages/support/resources/views/components/section.blade.php

```diff
@@ -139,14 +140,22 @@ class="-m-2"
             @if ($collapsed)
                 x-cloak
             @endif
-            x-bind:class="{ 'invisible h-0 border-none': isCollapsed }"
+            @if (!$hasAnimation){
+                x-bind:class="{ 'invisible h-0 border-none': isCollapsed }"
```

Comment:
There's a reason we use this approach, right @danharrin?
Won't this break animated collapsible sections?

Reply:
Break in what way?

Reply:
Possibly we didn't use `x-show` for a reason to do with form components state? Since `x-show` is used for this new animated collapsible section, I'm afraid that must break something?

Reply:
Can we animate the height property instead maybe

Reply:
> Can we animate the height property instead maybe

Well, technically you can do it with `max-height`, but in order to do so you need to set a hardcoded value. I didnt find any issues with `x-show` though and its used in the groups.php so why not go with `x-show`?

Reply:
I don't know the exact reason, but there is a reason for the current approach.